.devcontainer/Dockerfile

```diff
@@ -7,6 +7,7 @@ FROM mcr.microsoft.com/vscode/devcontainers/base:ubuntu-20.04
 # RUN bash -i -c 'pip3 install --user --no-use-pep517 -e .[all]'
 RUN apt-get update && \
     apt-get -y upgrade && \
+    apt-get install make && \
```

Comment:
Works great and is better than the syntax in my PR.  I will delete my PR:
https://github.com/HERMES-SOC/hermes_instrument/pull/13